docs/additional-functionality/qualification-profiling-tools.md

```diff
@@ -198,6 +198,14 @@ Usage: java -cp rapids-4-spark-tools_2.12-<version>.jar:$SPARK_HOME/jars/*
                                     use ~APPLICATION_NAME. i.e Select all event
                                     logs except the ones which have application
                                     name as the input string.
+      --any                         Apply multiple event log filtering criteria and process
+                                    only logs for which any condition is satisfied.
+                                    Example: <Filter1> <Filter2> <Filter3> --any -> result
+                                    is <Filter1> OR <Filter2> OR <Filter3>
+      --all                         Apply multiple event log filtering criteria and process
+                                    only logs for which all conditions are satisfied.
+                                    Example: <Filter1> <Filter2> <Filter3> --all -> result
+                                    is <Filter1> OR <Filter2> OR <Filter3>. Default is all=true.                              
```

Comment:
Shouldn't this be `<Filter1> AND <Filter2> AND <Filter3>` ?

Reply:
Yes. Copied from above but forgot to update here. Thanks for noticing this.